deployment/src/test/java/com/datastax/oss/quarkus/runtime/driver/UseQuarkusNettyEventLoopEnabledTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.runtime.driver;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.internal.core.context.NettyOptions;
+import com.datastax.oss.quarkus.CassandraTestBase;
+import io.quarkus.test.QuarkusUnitTest;
+import javax.inject.Inject;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class UseQuarkusNettyEventLoopEnabledTest {
+  @Inject CqlSession cqlSession;
+
+  @RegisterExtension
+  static QuarkusUnitTest runner =
+      new QuarkusUnitTest()
+          .setArchiveProducer(
+              () -> ShrinkWrap.create(JavaArchive.class).addClasses(CassandraTestBase.class))
+          .withConfigurationResource("application-quarkus-netty-enabled.properties");
+
+  @Test
+  public void should_use_quarkus_netty_event_loop() {
+    // when
+    NettyOptions nettyOptions = ((QuarkusDriverContext) cqlSession.getContext()).getNettyOptions();
+
+    // then
+    assertThat(nettyOptions).isInstanceOf(QuarkusNettyOptions.class);
```

Comment:
Maybe those test could go a little further: capture a driver I/O thread, using an async request callback, like the example in my previous comment. Then assert that it's from the expected event loop: you could use the thread names, or the fact that default driver threads implement `com.datastax.oss.driver.internal.core.util.concurrent.BlockingOperation.InternalThread`. Or both.